src/index.js

```diff
@@ -289,7 +298,7 @@ class Dropzone extends React.Component {
         onDragLeave={this.onDragLeave}
         onDrop={this.onDrop}
       >
-        {this.props.children}
+        {this.renderChildren()}
```

Comment:
If you're not using `this` in the method, you can extract it as a function outside of the class. Or you could use `static` keyword.